config/locales/en/legislation.yml

```diff
@@ -64,6 +64,16 @@ en:
         no_open_processes: There aren't open processes
         no_next_processes: There aren't planned processes
         no_past_processes: There aren't past processes
+        section_header:
+          icon_alt: Legislation processes icon
+          title: Legislation processes
+          description: Participates in the debates and processes prior to the approval of a ordinance or a municipal action. Your opinion will be consider by the City Council.
```

Comment:
`Take part in the debates and processes prior to the approval of and ordinance or municipal action. Your opinion will be consider by the City Council.`?